fenix/app/src/main/res/values/static_strings.xml

```diff
@@ -124,4 +124,21 @@
     <string name="translations_bottom_sheet_translate_to">Translate to</string>
     <string name="translations_bottom_sheet_negative_button">Not now</string>
     <string name="translations_bottom_sheet_positive_button">Translate</string>
+
+    <string name="translation_option_bottom_sheet_title">Translation Options</string>
+    <string name="translation_option_bottom_sheet_always_translate">Always offer to translate</string>
+    <string name="translation_option_bottom_sheet_always_translate_in_language">Always translate %1$s</string>
+    <string name="translation_option_bottom_sheet_never_translate_in_language">Never translate %1$s</string>
+    <string name="translation_option_bottom_sheet_never_translate_site">Never translate this site</string>
+    <string name="translation_option_bottom_sheet_translation_settings">Translation settings</string>
+    <string name="translation_option_bottom_sheet_about_translations">About translations in %1$s</string>
+
+    <string name="translation_settings_fragment_toolbar_title">Translations</string>
+    <string name="translation_settings_fragment_translation_preference">Translation preferences</string>
+    <string name="translation_settings_fragment_offer_to_translate">Offer to translate when possible</string>
+    <string name="translation_settings_fragment_always_download">Always download languages  in data saving mode</string>
+    <string name="translation_settings_fragment_automatic_translation">Automatic translation</string>
+    <string name="translation_settings_fragment_automatic_never_translate_site">Never translate this site</string>
```

Comment:
I think we need separate strings for the "Translations" page linked as "Translation settings". 

In the second page, it should be the plural of "Never translate these sites" via [this](https://www.figma.com/file/GT6oZLFw32LiCwKl5GOMWD/Full-Page-Translation?node-id=7202%3A113050&mode=dev). (This copy is [correct](https://www.figma.com/file/GT6oZLFw32LiCwKl5GOMWD/Full-Page-Translation?node-id=7202%3A99806&mode=dev) for the main "Translation Options" toggle.)

Reply:
Done